UPGRADE.md

```diff
@@ -23,6 +23,48 @@
 
 1. The constructor of `Sylius\RefundPlugin\Provider\DefaultRelatedPaymentIdProvider` has been removed.
 
+1. Constructor of `Sylius\RefundPlugin\Entity\CustomerBillingData` has been removed and now `CustomerBillingData` entity
+   is created by `Sylius\RefundPlugin\Factory\CustomerBillingDataFactory`.
+
+1. The constructor of `Sylius\RefundPlugin\Generator\CreditMemoGenerator` has been changed:
+
+    ```diff
+        public function __construct(
+            LineItemsConverterInterface $lineItemsConverter,
+            LineItemsConverterInterface $shipmentLineItemsConverter,
+            TaxItemsGeneratorInterface $taxItemsGenerator,
+            CreditMemoFactoryInterface $creditMemoFactory,
+    +       CustomerBillingDataFactoryInterface $customerBillingDataFactory
+        ) {
+            ...
+        }
+    ```
+
+1. The getter methods of `Sylius\RefundPlugin\Entity\CustomerBillingData` has been changed:
+
+    ```diff
+    -   public function id(): ?int
+    +   public function getId(): ?int
+    -   public function firstName(): string
+    +   public function getFirstName(): string
+    -   public function lastName(): string
+    +   public function getLastName(): string
+    -   public function fullName(): string
+    +   public function getFullName(): string
+    -   public function street(): string
+    +   public function getStreet(): string
+    -   public function postcode(): string
+    +   public function getPostcode(): string
+    -   public function city(): string
+    +   public function getCity(): string
+    -   public function company(): ?string
+    +   public function getCompany(): ?string
+    -   public function provinceName(): ?string
+    +   public function getProvinceName(): ?string
+    -   public function provinceCode(): ?string
+    +   public function getProvinceCode(): ?string
```

Comment:
I'm not sure if we should change these getters, probably the better way would be not changing them or adding new ones and leaving old methods with deprecation

Reply:
I have added deprecated annotations and leaved a note in file about upcoming changes 🙂